distributed_nose/plugin.py

```diff
@@ -107,13 +108,25 @@ def options(self, parser, env):
                 "using '--algorithm=least-processing-time'."
             ),
         )
+        parser.add_option(
```

Comment:
it this needed? it looks like DISTRIBUTED_HASH_BY_CLASS already controls this.

Reply:
yeah, probably I can stick to this param, in that case we just need to specify it explicitly in Jenkins job